doc/contrib/pr.rst

```diff
@@ -3,6 +3,20 @@
 PR workflow
 ===========
 
+The PR, short for Pull Request, is a way to merge code on Github, where the main Arbor-repo is hosted.
+
+.. _contribpr-issue:
+
+Issues
+------
+
+New features, bugfixes or other kinds of contributions ideally start their lives as an Issue on our
+`Issue tracker <https://github.com/arbor-sim/arbor/issues>`_. Having an Issue before an implementation
+addressing the Issue (code contribution or otherwise) gives others the chance to weigh in and help
```

Comment:
Now that we have explained that: replace `Isssue` with request here

Reply:
Again, maybe to subtly worded right now, but I want to stress that the thing being addressed exists as a Github Issue. Referring to them as requests is a layer of indirection that I would rather leave out, unless you have a strong argument for doing that anyway.

Reply:
The sentence as stated

> Having an Issue before an implementation addressing the Issue

is syntactically dubious -- at least. Without having an Issue one cannot address it.
Either the first or second `Issue` should refer to something different. I think you
mean this

> Make a (formal GH-kind) Issue before addressing your (desire-for-change-kind) issue (in code/text).

Reply:
I've updated with a definition (like there already was for PR) the first time the term is used.